### DISCUSSION TITLE
Schema Version

### DISCUSSION THREAD

**Discussion by HenokMillion** [Travelport-Ukraine/uapi-json#532] [General]
Thank you for the nice wrapper.
Can you please let us know if there is a plan to update the XML schema from 47 to the latest one?

**Comment by Smotrov** [comment]
Thank you for using this tool! 🥰
We do not have an exact plan for it. However, we would be happy accept your contribution. 
More details [here](https://github.com/Travelport-Ukraine/uapi-json/blob/master/CONTRIBUTING.md).
